purchase_delays/models/product_supplierinfo.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class ProductSupplierinfo(models.Model):
+
+    _inherit = "product.supplierinfo"
+
+    delay = fields.Integer(
+        string="Lead Time",
+        compute="_compute_delay",
+        inverse="_inverse_delay",
+        readonly=True,
+    )
+    supplier_delay = fields.Integer(
+        string="Supplier Lead Time", default=1, required=True
+    )
+    transport_delay = fields.Integer(
+        string="Transport Lead Time", default=0, required=True
+    )
+
+    @api.depends("supplier_delay", "transport_delay")
+    def _compute_delay(self):
+        for record in self:
+            record.delay = record.supplier_delay + record.transport_delay
+
+    def _inverse_delay(self):
```

Comment:
this inverse shouldn't be here, you shouldn't update fields on which you are computing this field

Reply:
Yeah, but the build is failing since `supplier_calendar` is trying to set `delay`

Reply:
What I can do is to create a new field instead.

Reply:
then you need to add to the description it's not compatible with supplier_calendar, and AFAIS the problem in the test not in the model, you may need to adapt the test

Reply:
@mmacross Not sure how to solve this technically, but on a functional side it is correct that if you use the supplier_calendar it should set the supplier_delay (not the delay) if you use this module. Because calendar is related to when your supplier works, not transport.

cc @i-vyshnevska @LoisRForgeFlow

Reply:
I have set that inverse function because of the tests that was crashing because they was trying to set delay.
I'm not sure it's necessary to keep it if I declare those two modules as incompatible.

Reply:
we don't need to be compatible with supplier_calendar but as it installed after the test will fail

Reply:
@i-vyshnevska Can you detail why it's bad to get an inver on delay? I personally think it's a good thing. What to I miss ? 
My proposal @mmequignon would be setting the delay would set the supplier delay and keep transport to 0.

Reply:
Real math here ;) If delay i set then:
 Supplier delay = delay - transport delay
 transport delay is kept

That's how I would do

Reply:
@jgrandguillaume  because delay is computed field based on two other delays, if inverse should update transport_delay, than an update of transport_delivery should update delay and goes to eternity, odoo framework prevent this on some level but this looks wrong for me
@mmequignon you need to add a flag on test on supplier_calendar -post_install

Reply:
Understood @i-vyshnevska  thanks. @mmequignon Will call you to discuss it.